test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/StoreTransactionTests.cs

```diff
@@ -135,7 +138,7 @@ await using (MemoryStream stream = _recyclableMemoryStreamManager.GetStream())
                     multiContent.Add(validByteContent);
                 }
 
-                using DicomWebResponse<DicomDataset> response = await _client.StoreAsync(multiContent);
+                using DicomWebResponse<DicomDataset> response = await _instancesManager.StoreAsync(multiContent, instanceId: DicomInstanceId.FromDicomFile(validFile));
```

Comment:
Why is providing instanceId necessary here?

Reply:
`DicomInstancesManager` need the id to know which dicom file to delete, unfortunately we couldn't infer it from input, so need additional parameter for it.

Reply:
Makes sense, thanks!